src/NServiceBus.AmazonSQS.sln

```diff
@@ -22,6 +22,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		..\.github\workflows\release.yml = ..\.github\workflows\release.yml
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NServiceBus.Transport.SQS.TransportTests.DoNotWrapOutgoingMessages", "NServiceBus.Transport.SQS.TransportTests.DoNotWrapOutgoingMessages\NServiceBus.Transport.SQS.TransportTests.DoNotWrapOutgoingMessages.csproj", "{ADB39452-CFA5-4F53-9024-FCEF78A8693E}"
```

Comment:
Why do we need a separate project for this?

Reply:
I think that using the "variant" approach that SQLP does might be a better approach, happy to jump on a call and talk it through

https://github.com/Particular/NServiceBus.Persistence.Sql/blob/master/src/SqlPersistence.PersistenceTests/SqlTestVariant.cs

Reply:
@andreasohlund, as far as I can tell, there isn't support for variants in transport tests

Reply:
Talked to @mauroservienti and there might be a way to get this working, I will do a spike